Nano.Net/Utils.cs

```diff
@@ -144,29 +144,31 @@ private static string EncodedAddressChecksum(byte[] publicKey)
             return EncodeNanoBase32(final.Reverse().ToArray(), false);
         }
 
-        public static string AddressFromPublicKey(byte[] publicKey)
+        public static string AddressFromPublicKey(byte[] publicKey, string prefix)
```

Comment:
All methods that have been modified to work with different prefixes should still work without explicitly choosing a prefix, using the nano_ prefix by default, in order to preserve backwards compatibility and to simplify usage

Reply:
Please check 280e65b and 2d56fe4.